core/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -258,9 +260,29 @@ private Component renderTitle(final Config config, final MapInfo map) {
       return header.colorIfAbsent(NamedTextColor.AQUA);
     }
 
-    final Component game = map.getGamemode();
-    if (game != null) {
-      return game.colorIfAbsent(NamedTextColor.AQUA);
+    final Component gamemode = map.getGamemode();
+    if (gamemode != null) {
+      return gamemode.colorIfAbsent(NamedTextColor.AQUA);
+    }
+
+    final Collection<Gamemode> gamemodes = map.getGamemodes();
+    if (!gamemodes.isEmpty()) {
+      Iterator<Gamemode> gamemodeIterator = gamemodes.iterator();
+      String firstgamemodeId = gamemodeIterator.next().getId();
+      if (gamemodeIterator.hasNext()) {
+        String secondgamemodeId = gamemodeIterator.next().getId();
+        Component firstComponent =
+            Component.translatable("gamemode." + firstgamemodeId + ".acronym");
+        Component secondComponent =
+            Component.translatable("gamemode." + secondgamemodeId + ".acronym");
+        return firstComponent
+            .append(Component.text(" & "))
+            .append(secondComponent)
+            .colorIfAbsent(NamedTextColor.AQUA);
+      } else {
+        return Component.translatable("gamemode." + firstgamemodeId + ".name")
+            .colorIfAbsent(NamedTextColor.AQUA);
+      }
     }
```

Comment:
This is a very odd way to do this, you're essentially doing an if-hell of if it's one, or if it's 2 gamemodes, ignoring any other case, additionally, you're using a hard-coded `&`, which has to be localized for different languages.

Consider a more iterative approach like the one with map tags below